releasetasks/templates/tc_update_verify.yml.tmpl

```diff
@@ -0,0 +1,51 @@
+{% set uv_buildername = "{}_{}_update_verify".format(platform, branch) %}
+{% set uv_totalchunks = 12 %}
+{% for chunk in range(uv_totalchunks) %}
+-
+    taskId: "{{ stableSlugId(uv_buildername) }}"
```

Comment:
do we wan to include the chunk so we have unique taskid's? https://github.com/mozilla/releasetasks/blob/master/releasetasks/templates/l10n.yml.tmpl#L10